ironfish/src/storage/database.test.ts

```diff
@@ -678,6 +719,12 @@ describe('Database', () => {
     })
 
     it('should not find entries with an off-by-one prefix and empty key', async () => {
+      const testPrefixKeyStore = db.addStore<BazSchema>({
+        name: 'PrefixKey',
+        keyEncoding: new BufferEncoding(),
+        valueEncoding: new StringEncoding(),
+      })
```

Comment:
Just moved this here since no one else should use this anyway, as indicated by the comment.